apps/omg_db/test/support/db_case.ex

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.DB.Case do
+  @moduledoc """
+  Defines the useful common setup for all `...PersistenceTests`:
+   - starts temporary file handler `:briefly`
+   - creates temp dir with that
+   - initializes the low-level LevelDB storage and starts the test DB server
+  """
+
+  use ExUnit.CaseTemplate
+
+  using do
```

Comment:
Wow, this's interestning! So by `use` this file in tests we have `setup_all` and `setup` callbacks fired by ExUnit :question: 
Nice!

Reply:
:thinking: it's possible that this empty using isn't necessary at all, I might have left it here by mistake

Reply:
oh, and on `ExUnit.CaseTemplate` - yes - this is the more "standard" way of doing what we're doing with `ex_unit_fixtures`, I wanted to try it out with the intention of preferring `ExUnit.CaseTemplate` wherever it's enough (`deffixture` offers a lot more in terms of composability of the steup routines)